overlord/snapstate/snapstate.go

```diff
@@ -179,6 +179,12 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		prev = setupSecurityPhase2
 	}
 
+	// auto-connections
+	autoConnect := st.NewTask("auto-connect", fmt.Sprintf(i18n.G("Automatically connect eligible plugs and slots of snap %q"), snapsup.Name()))
```

Comment:
+1 and following last discussion, this looks correct, placed after the 2nd setup-profiles if this is core and happens after link-snap